Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py

```diff
@@ -1,43 +1,97 @@
+import demistomock as demisto
 from CommonServerPython import *
 import json
+import re
 
+ERROR_TEMPLATE = 'ERROR: PreprocessEmail - {function_name}: {reason}'
+QUOTE_MARKERS = ['<div class="gmail_quote">',
+                 '<hr tabindex="-1" style="display:inline-block; width:98%"><div id="divRplyFwdMsg"']
 
-def add_entries(single_reply, email_related_incident):
+
+def create_email_html(email_html='', entry_id_list=[]):
```

Comment:
don't use mutable objects as a default value, this is a bug.
use None and handle that case inside the function.
https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/